.github/workflows/pr-checks.yml

```
@@ -30,7 +30,7 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - name: Cache node_modules
-        uses: actions/cache@v1
+        uses: actions/cache@v2
```

Joar, haste ja flonke gemacht!

---

```suggestion
        uses: actions/cache@v3
```
nimm mal lieber v3 Brüh